webapp/maps_heatmaps/test/declareDatabase.ts

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
What is this file for? Is duplicated and should be removed, no?

Reply:
I removed it.